frontend/server/src/Controllers/School.php

```diff
@@ -164,14 +164,20 @@ public static function createSchool(
     public static function apiSchoolCodersOfTheMonth(\OmegaUp\Request $r): array {
         $r->ensureInt('school_id');
         $school = \OmegaUp\DAO\Schools::getByPK(intval($r['school_id']));
-
+        \OmegaUp\Validators::validateOptionalInEnum(
+            $r['category'],
+            'category',
+            \OmegaUp\Controllers\User::ALLOWED_CODER_OF_THE_MONTH_CATEGORIES
+        );
+        $category = !empty($r['category']) ? $r['category'] : 'all';
```

Comment:
nit:

```suggestion
        $category = $r['category'] ?? : 'all';
```

igual en todos los demás lugares donde se usa este patrón en este cambio.

Reply:
listo